### PR TITLE
Remove restriction

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: docs/src/main/sphinx/optimizer/pushdown.rst [hunk 1]
```diff
@@ -75,7 +75,6 @@ operator. This shows you that the pushdown was successful.
 
 A number of factors can prevent a push down:
 
-* adding a condition to the query
```
Comment (findepi) [inline docs/src/main/sphinx/optimizer/pushdown.rst:78]: This is correct

**Review by sopel39 — APPROVED** [review]

**Review by findepi — CHANGES_REQUESTED** [review]
I think the removed wording was factually correct. Can you please add some more context why do you think we should remove it?

**Comment by mosabua** [comment]
Supposedly it was correct in the past but no longer applies.

**Comment by mosabua** [comment]
Turns out everything is as it should be after all

**Pull request closed without merging**